usr/share/lib/ipa/tests/SLES/EC2/test_sles_ec2_network.py

```diff
@@ -33,19 +33,22 @@ def test_sles_ec2_network(determine_region, host):
 
     region = determine_region('ec2')
 
+    if region == 'ap-northeast-3':
+        pytest.skip(
+            'Skipped special region: {0}'.format(region)
+        )
+
     url = 'https://suse-download-test-{0}.s3.amazonaws.com/' \
           'SLE-15-Installer-DVD-x86_64-GM-DVD2.iso'.format(
               region
           )
 
     dl_result = host.run(
-        'curl -o /dev/null -skw "%{{time_total}} %{{size_download}}"'
-        ' {0}'.format(url)
+        'curl -o /dev/null -m {0} -skw "%{{size_download}}" {1}'.format(
```

Comment:
Please use long options for the curl command. I don't think that any one of us is a frequent curl user such that we can tell what this does without looking at the man page. Especially the choice of '-k' == '--insecure' is cryptic.

Good find on the --max-time option